netdata/netdata
# Add label for cgroup

##### Summary
eBPF.plugin was adding label for `services`, but we did not have for `cgorups`. This PR address this issue.

##### Test Plan

The `cgroup_name` will appear only for cgroups that are not services, to show them you can do the following steps:

1. Install docker 
2. Run `docker run -it alpine /bin/sh`

Now

1. Compile this branch
3. Enable integration with cgroup:
```sh
[global]
    cgroups = yes
```
4. Start netdata
5. Access `http://localhost:19999/api/v1/charts?all` and confirm `cgroup_name` appears.

##### Additional Information


<details> <summary>For users: How does this change affect me?</summary>
  
</details>
